features/step_definitions/appointment_locator_steps.rb

```diff
@@ -1,11 +1,11 @@
 When(/^I search for appointment locations near to a valid postcode$/) do
-  locations = 1.upto(7).map { |x| Locations::Location.new("cab-#{x}", [1, 0]) }
+  locations = 1.upto(7).map { |x| Locations::Location.new("cab-#{x}", '', '', [1, 0]) }
```

Comment:
Maybe time to upgrade Location's initialisation? It would be preferable for address and hours to be optional instead of having to pass in a blank string.